delivery/client.py

```diff
@@ -43,3 +52,31 @@ def component_descriptor(
             res.json(),
             validation_mode=validation_mode,
         )
+
+    def post_compliance_scan(
+        self,
+        scan_data: dict,
+        compliancedb_cfg_name: str,
+        tool: ScanTool,
+        component_name: str,
+        component_version: str,
+        artifact_name: str,
+        artifact_version: str,
+        artifact_type: ArtifactType,
+    ):
+        body = {
```

Comment:
since you also need to deserialise this on the receiving endpoint, you should - as discussed before - add serialisation/deserialisation functions (this will come in handy for the PR w/ the delivery-service)